apps/ideaspace/src/components/modules/WorkshoppingPage/IdeaOverview/DescriptionCard/StyledComponents.js

```diff
@@ -0,0 +1,28 @@
+import { Card } from '../../../../common/Card/Card'
+import styled from '@emotion/styled';
+
+export const StyledCard = styled(Card)`
+  display: flex;
+  flex-direction: column;
+  padding: 20px;
+  padding-left: 30px;
+  padding-right: 30px;
+  margin-bottom: 25px;
+`;
+
+export const Cardheading = styled.div`
```

Comment:
could we change `Cardheading` into an `<h3>` to remove the nested `<h3>` elements in each of the `<Cardheading>` components?